src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -94,6 +115,8 @@ public static function register($mode = 0)
                     || in_array('legacy', \PHPUnit_Util_Test::getGroups($class, $method), true)
                 ) {
                     $group = 'legacy';
+                } elseif (DeprecationErrorHandler::MODE_WEAK_VENDORS === $mode && $isVendor) {
+                    $group = 'remaining vendor';
```

Comment:
would give the output "remaining from vendors deprecations". I think "remaining vendor deprecations" is better, right?